.gitignore

```diff
@@ -88,3 +88,4 @@ Network Trash Folder
 Temporary Items
 .apdisk
 
+.vscode
```

Comment:
I think it is not necessary. It is better for you to use a [global .gitignore file](https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) to ignore your editor files in every project you contribute, instead of having a huge file ignoring every possible editor.

Reply:
Yes! It really doesn't have to be there. Removed.